xtask/Cargo.toml

```diff
@@ -8,11 +8,11 @@ publish = false
 default = ["dep:isahc", "dep:semver", "dep:toml_edit"]
 
 [dependencies]
-clap = { version = "3.2.4", features = ["derive"] }
+clap = { version = "4.0.18", features = ["derive"] }
```

Comment:
Did we magically get compatible with clap v4? I remember there being some big major-version changes with it in the past.

Reply:
Not sure what the problem was before, but clearly it doesn't affect us now :)

Reply:
No concerns then if xtask is happy compiling it, thanks :)